packages/openapi3/src/openapi.ts

```diff
@@ -390,6 +397,9 @@ function createOAPIEmitter(program: Program, options: OpenAPIEmitterOptions) {
 
     contentEntry.schema = getSchemaOrRef(bodyModel);
 
+    // If no status code was defined in the response model, use 200.
+    statusCode ??= "200";
```

Comment:
How do you see this interacting with declarations using the proposed `@error` decorator?

Reply:
I think `@error` should make statusCode default to `default`.  And otherwise should add `x-ms-errror-response: true` to the response.